python/src/NAIS_doc.i.in

```diff
@@ -0,0 +1,126 @@
+%feature("docstring") OT::NAIS
+
+"Non Adaptive Importance Sampling (NAIS) algorithm.
+
+Parameters
+----------
+event : :class:`~openturns.RandomVector`
+    Event we are computing the probability of failure.
+
+rhoQuantile : float  :math:`0<\rho<1`
+    Intermediate quantile level.
+
+Notes
+-----
+
+The following explanations are given for a failure event defined as :math:`g(\mathbf{X})<T` with :math:`\mathbf{X}` a random vector following a joint PDF :math:`f_\mathbf{X}`, :math:`T` a threshold and :math:`g` a limit state function, without loss of generality.
+
+The Importance Sampling (IS) probability estimate by Importance Sampling :math:`\widehat{P}^\text{IS}` is given by:
+
+.. math::
+
+    \widehat{P}^\text{IS}=\frac{1}{N} \sum_{i=1}^{N} {\mathbf{1}}_{g(\mathbf{x}_i)<T} \frac{h_0(\mathbf{x}_i)}{h(\mathbf{x}_i)}.
+
+with :math:`h_0 = f_\mathbf{X}` the PDF of :math:`\mathbf{X}`, :math:`h` the auxiliary PDF of Importance Sampling, :math:`N`the number of independent samples generated with :math:`h`
+and :math:`{\mathbf{1}}_{g(\mathbf{x}_i)<T}` the indicator function of failure domain. 
+
+The optimal density minimizing the variance of the estimator :math:`h_{opt}` is  defined as:
+
+.. math::
+
+    h_{opt}=\frac{{\mathbf{1}}_{g(x)<T}h_0}{P}.
+
+with :math:`P` the failure probability which is inaccessible in practice since this probability is the quantity of interest and unknown. 
+
+The objective of Non parametric Adaptive Importance Sampling (NAIS) technique is to approximate the IS optimal auxiliary density :math:`h_{opt}` from the preceding equation with a kernel density function (e.g. Gaussian kernel). Its iterative principle is described by the following steps.
+
+1. :math:`k=1` and set :math:`\rho \in [0,1]`
+
+2. Generate the population :math:`\mathbf{x}_1^{(k)},...,\mathbf{x}_N^{(k)}` according to the PDF :math:`h_{k-1}`, apply the function :math:`g` in order to have :math:`y_1^{(k)}=g(\mathbf{x}_1^{(k)}),...,y_N^{(k)} = g(\mathbf{x}_N^{(k)})`
+
+3. Compute the empirical :math:`\rho`-quantile :math:`q_k=\max(T,y^{(k)}_{\left \lfloor \rho N \right\rfloor})`
+
+
+4. Estimate :math:`I_k= \frac{1}{kN} \displaystyle \sum_{j=1}^{k}\sum_{i=1}^{N} {\mathbf{1}}_{g(\mathbf{x}_i^{(j)}) \leq q_k} \frac{h_0(\mathbf{x}_i^{(j)})}{h_{j-1}(\mathbf{x}_i^{(j)})}` 
+
+5. Update the Gaussian kernel sampling PDF with:
+
+.. math::
+
+    h_{k}(\mathbf{x})=\frac{1}{k N I_k \det\left(B_{k+1}\right)}\sum_{j=1}^{k}\sum_{i=1}^{N}  w_{j}(\mathbf{x}_i^{(j)})K_d\left(B_{k+1}^{-1}\left(\mathbf{x}-\mathbf{x}_i^{(j)}\right)\right)
+
+where :math:`K_d` is the standard :math:`d`-dimensional Gaussian function with zero mean and a diagonal covariance matrix. 
```

Comment:
```suggestion
where :math:`K_d` is the standard :math:`d`-dimensional normal PDF with zero mean and identity covariance matrix,
```
I assume this is the case?

Reply:
ok i put normal distribution and leave diagonal covariance matrix because it is not necessarily a identity matrix

Reply:
3 remarks:

- I guess K_d is a normal **PDF**, not a normal distribution.
- If the normal distribution is "standard", then the covariance matrix is the identity matrix. Otherwise it cannot be called "standard".
- How do you choose the coefficients of the covariance matrix of the normal distribution K_d is the PDF of? You don't mention this in the doc.